app/Console/Commands/SyncJoindInEvents.php

```diff
@@ -30,11 +30,17 @@ public function handle()
     {
         $this->info('Syncing events...');
 
-        foreach ($this->client->getEvents() as $event) {
-            $this->info('Creating/updating event ' . $event['name']);
-            $this->importer->import($event['id']);
-        }
+        try {
+            $events = $this->client->getEvents();
 
-        $this->info('Events synced.');
+            foreach ($events as $event) {
+                $this->info('Creating/updating event ' . $event['name']);
+                $this->importer->import($event['id']);
+            }
+
+            $this->info('Events synced.');
+        } catch (Exception $exception) {
```

Comment:
Are we intentionally catching exceptions on lines 36-39? If not, would it be better in any way to just try around line 34? I'm never 100% sure how I should handle this.

Reply:
I had the same thought. I think it depends. In this case, the catch would also grab any thrown exceptions from calling the importer. I don't necessarily think that's what we want. Since we only want to silence the github client calls. I'll fix this up.

Reply:
I meant the JoindIn client calls.